

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Pretrained_models &mdash; atacworks v0.3.0 documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../_static/jquery.js"></script>
        <script type="text/javascript" src="../_static/underscore.js"></script>
        <script type="text/javascript" src="../_static/doctools.js"></script>
        <script type="text/javascript" src="../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Training an AtacWorks model" href="tutorial1.html" />
    <link rel="prev" title="AtacWorks" href="../readme.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../index.html" class="icon icon-home"> atacworks
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../readme.html">AtacWorks</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Pretrained_models</a></li>
<li class="toctree-l1"><a class="reference internal" href="tutorial1.html">Training an AtacWorks model</a></li>
<li class="toctree-l1"><a class="reference internal" href="tutorial2.html">Denoising and peak calling using a trained AtacWorks model</a></li>
<li class="toctree-l1"><a class="reference internal" href="../Dockerfile.html">AtacWorks Dockerfile</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../index.html">atacworks</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../index.html">Docs</a> &raquo;</li>
        
      <li>Pretrained_models</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../_sources/tutorials/pretrained_models.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="pretrained-models">
<h1>Pretrained_models<a class="headerlink" href="#pretrained-models" title="Permalink to this headline">¶</a></h1>
<p>This is a list of AtacWorks denoising models that were reported in Lal &amp; Chiang, 2019.
Models were trained to denoise bulk or single-cell ATAC-seq data of varying sequencing depths.</p>
<p>Note that all the AtacWorks models described in here are available to <a class="reference external" href="https://ngc.nvidia.com/catalog/models">download</a>.</p>
<p>To download a model and it’s corresponding data, search for the name at “Model path” column in the NGC model catalog. You can find a wget command at the bottom of the description box of the model as shown in the screenshot below.</p>
<a class="reference internal image-reference" href="../_images/ngc_catalog_screenshot.png"><img alt="../_images/ngc_catalog_screenshot.png" src="../_images/ngc_catalog_screenshot.png" style="width: 800px;" /></a>
<p>To view all the files, click on the <cite>File Browser</cite> tab. You can download individual files by clicking on a subfloder then clicking on the three dots alongside the file you would like to download.</p>
<a class="reference internal image-reference" href="../_images/ngc_file_download.png"><img alt="../_images/ngc_file_download.png" src="../_images/ngc_file_download.png" style="width: 800px;" /></a>
<p>The training data for these models was generated from BAM files using <a class="reference external" href="https://github.com/zchiang/atacworks_analysis/blob/master/preprocessing/atac_bam2bw.sh">this script</a>.</p>
<p>Briefly, each ATAC-seq read was converted to a single genomic position corresponding to the first base pair of the read. Reads aligning to the + strand were offset by +4 bp, while reads aligning to the - strand were offset by -5 bp. Each cut site location was extended by 100 bp in either direction. The bedtools genomecov function was used to convert the list of locations into a coverage track. To call peaks from clean and noisy signal tracks, MACS2 subcommands bdgcmp and bdgpeakcall were run with the ppois parameter and a -log10(p-value) cutoff of 3. BED files with equal coverage over all chromosomes were provided as a control input track.</p>
<p>Before using these models, please note that they should only be used on data that is processed exactly as described above. If your bigWig files and/or peak calls are generated using a different method, these models may give unreliable results, and it would be preferable to train your own model as described in Tutorial 1.</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 7%" />
<col style="width: 9%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 7%" />
<col style="width: 13%" />
<col style="width: 11%" />
<col style="width: 28%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Data type</p></th>
<th class="head"><p>Noise type</p></th>
<th class="head"><p>Noisy data
depth</p></th>
<th class="head"><p>Clean data
depth</p></th>
<th class="head"><p>Noisy data
cell count</p></th>
<th class="head"><p>Clean data
cell count</p></th>
<th class="head"><p>Single-cell
protocol</p></th>
<th class="head"><p>Training cell type(s)</p></th>
<th class="head"><p>Interval size (bp)</p></th>
<th class="head"><p>Model path</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Bulk</p></td>
<td><p>Low coverage</p></td>
<td><p>0.2M</p></td>
<td><p>50M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>CD4, CD8, B, NK</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowcov_0_2m_50m</p></td>
</tr>
<tr class="row-odd"><td><p>Bulk</p></td>
<td><p>Low coverage</p></td>
<td><p>1M</p></td>
<td><p>50M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>CD4, CD8, B, NK</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowcov_1m_50m</p></td>
</tr>
<tr class="row-even"><td><p>Bulk</p></td>
<td><p>Low coverage</p></td>
<td><p>5M</p></td>
<td><p>50M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>CD4, CD8, B, NK</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowcov_5m_50m</p></td>
</tr>
<tr class="row-odd"><td><p>Bulk</p></td>
<td><p>Low coverage</p></td>
<td><p>10M</p></td>
<td><p>50M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>CD4, CD8, B, NK</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowcov_10m_50m</p></td>
</tr>
<tr class="row-even"><td><p>Bulk</p></td>
<td><p>Low coverage</p></td>
<td><p>20M</p></td>
<td><p>50M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>CD4, CD8, B, NK</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowcov_20m_50m</p></td>
</tr>
<tr class="row-odd"><td><p>Bulk</p></td>
<td><p>Low quality</p></td>
<td><p>20M</p></td>
<td><p>20M</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>N/A</p></td>
<td><p>Monocytes</p></td>
<td><p>50,000</p></td>
<td><p>atac_bulk_lowqual_20m_20m</p></td>
</tr>
<tr class="row-even"><td><p>Single-cell</p></td>
<td><p>Low cell count</p></td>
<td><p>~0.2M</p></td>
<td><p>~13M</p></td>
<td><p>90</p></td>
<td><p>6000</p></td>
<td><p>dsci-ATAC</p></td>
<td><p>CD4, CD8, preB</p></td>
<td><p>50,000</p></td>
<td><p>atac_dsci_atac_lowcellcount_0_2m_13m_90_6000</p></td>
</tr>
<tr class="row-odd"><td><p>Single-cell</p></td>
<td><p>Low cell count</p></td>
<td><p>~1M</p></td>
<td><p>~13M</p></td>
<td><p>450</p></td>
<td><p>6000</p></td>
<td><p>dsci-ATAC</p></td>
<td><p>CD4, CD8, preB</p></td>
<td><p>50,000</p></td>
<td><p>atac_dsci_atac_lowcellcount_1m_13m_450_6000</p></td>
</tr>
<tr class="row-even"><td><p>Single-cell</p></td>
<td><p>Low cell count</p></td>
<td><p>~0.2M</p></td>
<td><p>~48M</p></td>
<td><p>10</p></td>
<td><p>2400</p></td>
<td><p>dsc-ATAC</p></td>
<td><p>B, Monocytes</p></td>
<td><p>50,000</p></td>
<td><p>atac_dsci_atac_lowcellcount_0_2m_48m_10_2400</p></td>
</tr>
<tr class="row-odd"><td><p>Single-cell</p></td>
<td><p>Low cell count</p></td>
<td><p>~1M</p></td>
<td><p>~48M</p></td>
<td><p>50</p></td>
<td><p>2400</p></td>
<td><p>dsc-ATAC</p></td>
<td><p>B, Monocytes</p></td>
<td><p>50,000</p></td>
<td><p>atac_dsci_atac_lowcellcount_1m_48m_50_2400</p></td>
</tr>
</tbody>
</table>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="tutorial1.html" class="btn btn-neutral float-right" title="Training an AtacWorks model" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="../readme.html" class="btn btn-neutral float-left" title="AtacWorks" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2020, NVIDIA Corporation

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>